streams/src/test/java/org/apache/kafka/streams/integration/utils/EmbeddedKafkaCluster.java

```diff
@@ -102,13 +101,14 @@ public EmbeddedKafkaCluster(final int numBrokers,
     /**
      * Creates and starts a Kafka cluster.
      */
-    public void start() throws IOException {
+    public void start() throws Exception {
         log.debug("Initiating embedded Kafka cluster startup");
-        log.debug("Starting a ZooKeeper instance");
-        zookeeper = new EmbeddedZookeeper();
-        log.debug("ZooKeeper instance is running at {}", zKConnectString());
+        final KafkaClusterTestKit.Builder clusterBuilder = new KafkaClusterTestKit.Builder(
+                new TestKitNodes.Builder()
+                        .setNumControllerNodes(1)
```

Comment:
Any reason why we don't setup the cluster with combined nodes and number of controllers == number of brokers instead of one controller and one broker? Specially that as far as I can see all the test set number of brokers to 1 anyway. This also can get us to remove `KafkaEmbedded` which might simplify the setup! WDYT?